UpdateDependencies.ps1

```diff
@@ -16,7 +16,7 @@ param(
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
-    [string]$GitHubPullRequestNotifications='')
+    [string]$GitHubPullRequestNotifications='stephentoub;ericstj;ellismg;weshaggard;joshfree')
```

Comment:
I think GitHub also has the ability to notify @ someRandomTeam / GitHub security group so the aliases don't have to be checked in a hardcoded list